modules/metafield_templates/languages/en.json

```diff
@@ -0,0 +1,18 @@
+{
+    "METAFIELD_TEMPLATES": {
+        "MODULE_DETAILS": {
+            "SINGULAR_NAME": "Metafield Template",
+            "PLURAL_NAME": "Metafield Templates"
+        },
+        "FIELDS": {
```

Comment:
Add Japanese (ja.json), Korean (ko.json), and Simplified Chinese (zh_cn.json) JSONs `Metafield Templates` module